src/mame/itech/itech32.h

```diff
@@ -20,12 +20,7 @@
 #include "emupal.h"
 #include "screen.h"
 
-#define VIDEO_CLOCK     XTAL(8'000'000)           // video (pixel) clock
-#define CPU_CLOCK       XTAL(12'000'000)          // clock for 68000-based systems
-#define CPU020_CLOCK    XTAL(25'000'000)          // clock for 68EC020-based systems
-#define SOUND_CLOCK     XTAL(16'000'000)          // clock for sound board
-#define TMS_CLOCK       XTAL(40'000'000)          // TMS320C31 clocks on drivedge
-
+#define LOG_DRIVEDGE_UNINIT_RAM     0
```

Comment:
Is there a specific reason you moved this macro to the header?  It only seems to be used in one file.  I’d understand if you were going to use it to avoid creating the `m_written` member at all if it’s disabled or something.  But it doesn’t seem to be used in the header at all.

Reply:
That's for avoid to create RAM logging related variables and functions.